### PR TITLE
Add area-System.Formats.Nrbf to area owners file.

### PR DESCRIPTION
**Pull request by eiriktsarpalis**

### CODE REVIEW

**Review by GrabYourPitchforks — APPROVED** [review]
My excitement is palpable. :)

**Comment by eiriktsarpalis** [comment]
Checking  if the tag works: @dotnet/area-system-formats-nrbf

File: docs/area-owners.md [hunk 1]
```diff
@@ -73,9 +73,9 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
+| area-System.Data.Odbc                          | @sammonort          | @ajcvickers @roji                                   |                                                                                                                                                                                                                                                                                            |
+| area-System.Data.OleDB                         | @sammonort          | @ajcvickers @roji                                   |                                                                                                                                                                                                                                                                                            |
```
Comment (bartonjs) [inline docs/area-owners.md:78]: "Fix whitespace" seems to have deleted spaces out of 3 these lines unnecessarily?

Comment (eiriktsarpalis) [reply docs/area-owners.md:78]: It was intentional, the vertical bars in these three lines were out of alignment.

File: docs/area-owners.md [hunk 2]
```diff
@@ -93,6 +93,7 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
 | area-System.Formats.Cbor                       | @jeffhandley         | @dotnet/area-system-formats-cbor                    | Consultants: @bartonjs @vcsjones                                                                                                                                                                                                                                                          |
+| area-System.Formats.Nrbf                       | @jeffhandley         | @dotnet/area-system-formats-nrbf                    | Consultants: @bartonjs @grabyourpitchforks                                                                                                                                                                                                                                                |
```
Comment (ericstj) [inline docs/area-owners.md:96]: Nit: looks like @bartonjs is a member of the team so he doesn't need to be listed as consultant.

Comment (bartonjs) [reply docs/area-owners.md:96]: Yeah, but he's probably going to try to get out of the team after not too long; but is still curious enough to lurk... so he doesn't have a problem with it.

Hmm, he's talking about himself in the 3rd person again...

**Pull request merged**